subprojects/plugin-use/src/main/java/org/gradle/plugin/use/resolve/internal/ArtifactRepositoryPluginResolver.java

```diff
@@ -46,7 +46,7 @@ public void resolve(final InternalPluginRequest pluginRequest, PluginResolutionR
             return;
         }
 
-        if (!pluginRequest.isConfigured()) {
+        if (!pluginRequest.isEnriched()) {
```

Comment:
The logic in the resolver is the same, no matter where the information in the request came from.